packages/react-core/src/components/Icon/Icon.tsx

```diff
@@ -11,11 +28,11 @@ export interface IconComponentProps extends Omit<React.HTMLProps<HTMLSpanElement
   /** Additional classes applied to the icon container */
   className?: string;
   /** Size of the icon component container and icon. */
-  size?: 'sm' | 'md' | 'lg' | 'xl';
+  size?: IconSize;
```

Comment:
don't we typically include the strings here so that it gets documented properly by the parser for the website?

Reply:
oh, was the parser changed to handle this?
![image](https://github.com/patternfly/patternfly-react/assets/5322142/852afdd7-ecec-4a9e-afa4-27a548203627)